crates/lib/src/helpers/path_ext.rs

```diff
@@ -1,8 +1,9 @@
-//! Shamelessly adapted from:
-//! https://github.com/rust-lang/cargo/blob/fede83ccf973457de319ba6fa0e36ead454d2e20/src/cargo/util/paths.rs#L61
+// Adapted from https://github.com/rust-lang/cargo/blob/fede83ccf973457de319ba6fa0e36ead454d2e20/src/cargo/util/paths.rs#L61
```

Comment:
Would it be a good idea to split that into a separate repository under cargo-bins?

Reply:
Already done!